src/tests/ftest/util/test_utils.py

```diff
@@ -185,6 +188,104 @@ def create(self):
             "  Pool created with uuid %s and svc ranks %s",
             self.uuid, self.svc_ranks)
 
+    @fail_on(DaosApiError)
+    def create_dmg(self, dmg_bin_path):
```

Comment:
I think your point is valid and there are pros and cons. I think it's safer not to touch the working create and create the new method. I'll rename `create` to ` create_dmg` in the tests.

Reply:
In my opinion, it would be ideal for there to only be one `create()` method which would conditionally call `create_with_api()` or `create_with_dmg()` based upon a new `self.use_dmg` conditional attribute (or whatever we want to call it - it should probably be a BasicParameter too).  This would prevent the need to change a whole bunch of tests and enable explicit calling of functions (if needed).

Reply:
Okay. I created `create_with_api` and `create_with_dmg`, and call it from `create`.